utils/cli/src/google/build.ts

```diff
@@ -110,6 +110,15 @@ const build = async (id: string, opts: BuildOptions) => {
 
 	// Write package.json
 	await packageJson(metadata, opts.dir);
+
+	// Write metadata.scss
+	await fs.writeFile(
```

Comment:
Minor nitpick, move the files to be written before `package.json`. Maybe before or after the README?

We use `package.json` to verify if a package needs to be rebuilt or updated. If this function fails halfway, then you may have a package that may be half written but still pass and not rebuild because `package.json` exists. 

Not an issue in CI since the build will fail, but developers working locally with the CLI might run into issues.

Reply:
Probably a good idea to add a comment explaining `package.json` should always be written last for future reference 😅